Lib/test/datetimetester.py

```diff
@@ -5916,6 +5916,65 @@ def test_timezones_capi(self):
 
                 self.assertEqual(dt1.astimezone(timezone.utc), dt_utc)
 
+    def test_get_delta_object_macros(self):
```

Comment:
Would you mind to also rename the test methods? Like test_PyDateTime_DELTA_GET().